### DISCUSSION TITLE
App's packages created by quasar-cli is not updated

### DISCUSSION THREAD

**Discussion by bangdragon** [quasarframework/quasar#10514] [Ideas / Proposals]
When create new app using quasar cli does not update the packages like: quasar, @quasar/extra
I've already updated the quasar-cli to the newest version 1.2.1

**Comment by ontwikkelfabriek** [comment]
@bangdragon 

> When create new app using quasar cli does not update the packages like: quasar, @quasar/extra

What do you mean with this?  When you create a new Quasar with the cli , **after** you run `npm/yarn install` it will use the `latest` Quasar libs available.

**Reply by bangdragon** [reply]
No, the new created quasar project use the old quasar libs: quasar 2.0.0
```
  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "axios": "^0.21.1",
    "core-js": "^3.6.5",
    "firebase": "^9.0.0",
    "quasar": "^2.0.0",
    "vue-i18n": "^9.0.0-beta.0",
    "vuex": "^4.0.1"
  },
```
While the current released libs are:
```
    "@quasar/extras": "^1.10.12",
    "quasar": "^2.0.4",
```

**Comment by ontwikkelfabriek** [comment]
@bangdragon 

> the new created quasar project use the old quasar libs: quasar 2.0.0

No, it does **not** say to use version 2.0.0.  This: **^2.0.0** means it will use a **compatible** version relative to 2.0.0 (usually the latest 2.0.x) . The exact version is stored ( after install of node_modules)  in **package.lock** or **yarn.lock**  ( depending if you use yarn or npm).

See:
https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

**Reply by bangdragon** [reply]
Oh, it's my fault. Now i got it. Thanks so much
